Minimized the sync problem

henpe071 opened the pull request:
Moved some signals around to minimize the sync problems. Also, clicking a drawing from the drawing widget now sets the frame to fit screen.

Remove the state as a pointer as it made some problems to the normal tags and when the state was sent around. The state a tagframe was holding was updated as it was sent around. What exactly was sent wasn't certain in all places.
Removed the pointers and the tags seems to work again and the tag sequence deletion that the pointers were supposed to fix also seems to work now. This might need some more through testing with all tags variations and deletions as this was fixed quickly just before the clock became too much.

henpe071 committed:
Minimized the sync problem

Moved some signals around to minimize the sync problems. Also, clicking a drawing from the drawing widget now set the frame to fit screen.

And remove the state as a pointer it made some problems to the normal tags and when the state was sent around.

niklasniklas commented:
Panning in image-seq-tag is not saved. Always centre
Tags in video does not work i development...

henpe071 committed:
Fix panning not working

The center point was not saved correctly so it used the default which would center the image

niklasniklas commented:
- [x] Pressing the label "Sequence" for a image tag seq changes the first tag state to fit-to-screen

- [x] Issue whne deleting an image in an image seq. The drawings in the "Drawings" widget does not get updated which leads to strange behavior

- [x] Shortcut "T" still works even if image-tag-seq

- [ ] Genrated report closes when Vian is closed (Save for later?)

- [x] Thumbnails in project window seems to be mixed RGB to BGR? They are saved in _thumbnail-folder?

- [x] Tag drawings doesnt seem to be working

henpe071 committed:
If video item is seq dont set state

henpe071 committed:
Turn of shortcut T when tag btn is disabled

henpe071 committed:
Drawing tag works again

niklasniklas commented:
- [x] When an analysis is done and you navigate to a POI. THen clicking the main video (parent to analysis) it jumps to a different frame

henpe071 committed:
Removing a frame from a sequence will now update the drawing widget and overlay

henpe071 committed:
Now works with normal sequences too

henpe071 committed:
Fix rgb convertion

henpe071 committed:
Navigating on analysis item now updates the state

henpe071 committed:
Fix context menu on drawing tag frame
